src/components/modal/v2/parts/ContentWrapper.jsx

```diff
@@ -35,6 +37,8 @@ const ContentWrapper = () => {
                         logo="logo"
                         headline={headline}
                         subheadline={subheadline}
+                        isQualifying={isQualifying}
```

Comment:
```suggestion
                        isQualifying={isQualifying ?? ''}
```
Another safety check here.